lib/common_test/src/test_server_ctrl.erl

```diff
@@ -3134,8 +3137,9 @@ do_get_data_dir(Mod,File) ->
 print_conf_time(0) ->
     ok;
 print_conf_time(ConfTime) ->
-    print(major, "=group_time    ~.3fs", [ConfTime]),
-    print(minor, "~n=== Total execution time of group: ~.3fs~n", [ConfTime]).
+	print(major, "=group_time    ~.3fs", [ConfTime]),
+	print(minor, "~n=== Total execution time of group: ~.3fs~n", [ConfTime]).
```

Comment:
- indentation pls fix here
- globally make sure you're not introducing unwanted tab characters - it is all over PR